test/integration/ctl/ctl_convert_test.go

```diff
@@ -89,6 +90,29 @@ func TestCtlConvert(t *testing.T) {
 			expOutput: `
 apiVersion: cert-manager.io/v1alpha3
 kind: Certificate
+metadata:
+  creationTimestamp: null
+  name: ca-issuer
+  namespace: sandbox
+spec:
+  commonName: my-csi-app
+  isCA: true
+  issuerRef:
+    group: cert-manager.io
+    kind: Issuer
+    name: selfsigned-issuer
+  secretName: ca-key-pair
+  subject:
+    organizations:
+    - hello world
+status: {}`,
+		},
+		"an object in v1alpha2 that uses a field that has been renamed in v1beta1 should be converted properly": {
+			input:         testdataResourceWithOrganizationV1alpha2,
+			targetVersion: targetv1beta1,
+			expOutput: `
+apiVersion: cert-manager.io/v1beta1
+kind: Certificate
```

Comment:
The output below looks identical to the output for the v1alpha2 > v1alpha3 conversion test above. Shall we use an example here which has some change between v1alpha3 and v1beta1 ? Or am I missing it?

Reply:
We convert from `v1alpha2` to `v1beta1`, so there is still a distinct change between the two API versions so there's not really a need to use a conversion that is _only_ present in v1beta1 and not in v1alpha3. v1alpha3 isn't involved in this test case.